runhouse/servers/http/http_utils.py

```diff
@@ -26,6 +27,12 @@ class ServerSettings(BaseModel):
     flush_auth_cache: Optional[bool] = None
 
 
+class PutResourceParams(BaseModel):
+    serialized_data: Any
+    serialization: Optional[str] = None
+    env_name: Optional[str] = None
```

Comment:
Just noting this is quite generic. I'd expect that if we had pydantic types for each route we'd go more specific (e.g. not bundle into serialized_data), or we reuse a type like this for several things which share these inputs.

Reply:
It'd be harder to make the serialization arg general then